code/analysis/analysis_civil/figures.do

```diff
@@ -447,44 +453,55 @@ if `event' == 1 {
 		#delimit ;
 		coefplot (TWFE, omitted baselevel msize(medlarge)), vertical
 			ciopts(recast(rcap) lwidth(.4) color(orange_red)) 
-			yline(0, lp(dash))
+			yline(0, lp(dash)) // yline(`att', lcolor(grey) lwidth(medium) lp(dash))
 			ylabel(-0.1(0.05)0.25)
 			xline(7.5)
 			xtitle("Years relative to treatment", size(medium))
 			ytitle("Effect of MeToo on `y'", size(medium))
-			`xlabel'
+			`xlabel' 
+			text(0.19 2 "ATT: `att'", size(medsmall))
```

Comment:
shouldn't they both be size(medsmall) color(black) if the relief_scale one is? or is automatically showing up as black?